src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,48 @@
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestState: function (id, url) {
```

Comment:
The id is only passed along to identify the request, you can use whatever you want to reference the it.